src/teleop/gui/README.md

```diff
@@ -0,0 +1,94 @@
+# GUI LCM Documentation 
+
+### Base Station GUI LCMs
+
+#### All GUIs:
```

Comment:
Probably delete the README.md or at least make sure that it's currently consistent with the current code. If you want to keep the template, just make it explicit that it's an example.

The last thing we want is for people to read documentation that does not represent current code

Reply:
I'll delete this at some point, just a remnant that I forgot to get rid of.

Reply:
Sounds good!